package.json

```diff
@@ -18,7 +20,9 @@
     "start": "react-scripts start",
     "build": "react-scripts build",
     "test": "react-scripts test",
-    "eject": "react-scripts eject"
+    "eject": "react-scripts eject",
+    "predeploy": "npm run build",
+    "deploy": "gh-pages -d build"
```

Comment:
[Optional] You can choose to combine pre-deploy script with deploy scripts with one command. using 
 ``` 
"deploy": "npm run build && gh-pages -d build"
```
Fasten your deployment time 🚀 😃